yafsrc/YetAnotherForum.NET/Pages/Admin/EditUser/UsersSettings.cshtml.cs

```diff
@@ -98,8 +98,8 @@ public IActionResult OnGet(int userId)
 
         this.Input = new UsersSettingsInputModel
         {
-                         UserId = userId
-                     };
+            UserId = userId,
```

Comment:
this would throw an error "Remove trailing comma...."

Reply:
I can remove it if you like, but the trailing comma is perfectly fine in c#. 
extra trailing commas are only a problem in json as far as I'm aware.

Reply:
Technically it works with c# and json. But it would still throw errors with code styling.